core/cmd/local_client.go

```diff
@@ -20,6 +21,10 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+// Bitwise representation of the permissions which are * not * permitted on files.
+// These are read, write, and execute permissions for groups and public
+const invalidPerms = os.FileMode(0077)
```

Comment:
I agree with @felder-cl that it's probably a little clearer to call this `invalidPermsMask`.  Otherwise a quick glance lends the impression that `---rwxrwx` is the only invalid perm.

Reply:
What do you think of c49c8c17c352a81b470618851f11ef9f43b03ed4

Reply:
Also the error Message should mention the what the permissions should be. It will  be friendly to the user.